protocol/vmutil/builder_test.go

```diff
@@ -0,0 +1,161 @@
+package vmutil
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"testing"
+
+	"chain/protocol/vm"
+)
+
+func TestAddInt64(t *testing.T) {
```

Comment:
Fine to leave it in, but is there a deeper reason for including this test with this batch?